agent/definitions.py

```diff
@@ -48,7 +48,7 @@ class VulnerabilityMetadata:
     title: str
     description: str
     reference: str
-    risk_rating: str
+    risk_rating: str = "CRITICAL"
```

Comment:
We default risk rating to `CRITICAL`??

Reply:
It was hardcoded in webexploit.WebExploit

Reply:
yes that's what we said